games/tic-tac-toe/TicTacToe.java

```diff
@@ -0,0 +1,40 @@
+import java.util.*;
+
+class TicTacToe {
+
+   public static void main(String[] args) {
+    Board board = null;
+    int size = 0;
+    int count = 0;
+    boolean isCompleted = false;
+
+    Scanner scanner = new Scanner(System.in);
+    do {
+        System.out.print("Please initialize the size greater than 2 ");
+        size = scanner.nextInt();
+        if (size > 2) {
+          board = new Board(size);
+        }
+      } while (board == null);
+    do{
+        count++;
```

Comment:
nit : Please as close to the first use as possible. This makes it easier for the reader to find the declaration and see what type the variable is and what it was initialized to.